### ISSUE TITLE
a little question about paper

### ISSUE THREAD

**Issue by feather820** [bbepoch/HoiTransformer#3]
It's a grate job, Tanks for what you contribute.
I have a question about the paper. In the paper, you said "All one-layer MLP branches for predicting confidence use a softmax function.". But as far as I know, VCOCO dataset's verbs is multi label, I am wonder how can I use softmax to predict multi label. 
Hope you can reply me
Thanks

**Comment by bbepoch** [comment]
Mainly because the one-to-one matching strategy and the definition of HOI instance. For example, given an image, their is a person sitting on a horse, and the GT contains 3 HOI categories, e.g. (person, ride, horse), (person, sit_on, horse), (person, straddle, horse). 

In training, the model directly outputs 100 HOI instances, the ground truth size is also 100 with 3 positive instances and 97 null padding. That is to say, there are only 3 slots responsible for the positive predictions.

In inference, if given this image, only 3 of the 100 slots are actived, so, after filterring, the model only outputs 3 HOI instances.

To summary, in the set prediction view, we don't think the original GT is one element with 3 labels, but 3 different elements.

The 'Appendix A: Illustration of Inference Process' part may help.

I hope it helped you, thanks.

**Comment by feather820** [comment]
Thanks, it helps me a lot
